src/reader.rs

```diff
@@ -1627,6 +1696,64 @@ impl<Term: Terminal> Reader<Term> {
         Ok(())
     }
 
+    /// Gets the current log receiver, or `Ok(None)` if one doesn't exist, or all senders have
+    /// been closed.
+    fn get_receiver(&mut self) -> Option<&LogReceiver> {
+        let drop_channel = if let Some(ref mut receiver) = self.log_channel {
```

Comment:
Hm. If the last sender were dropped before some messages were received and processed, wouldn't this cause those messages to be ignored?